src/pages/guides/index.md

```diff
@@ -251,7 +251,8 @@ This feature is scheduled to be rolled out on 10th February, 2022
 For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
 
 - sending additional field of recipient_client_id as part of your event payload
```

Comment:
Missed this in the previous review but let's code styling to the `recipient_client_id` field.
```suggestion
- sending additional field of `recipient_client_id` as part of your event payload
```

Reply:
done